qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -37,24 +37,9 @@ class InterleavedRB(StandardRB):
         the ground state, fits the two exponentially decaying curves, and estimates
         the interleaved gate error. See Ref. [1] for details.
 
-        See :class:`InterleavedRBAnalysis` documentation for additional
-        information on interleaved RB experiment analysis.
```

Comment:
Do we really want to remove this sentence? Maybe it can be helpful.

Reply:
Do we need this? Because another dedicated section for analysis class reference is generated.